digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/SpeedLimitService.scala

```diff
@@ -279,13 +304,13 @@ class SpeedLimitService(eventbus: DigiroadEventBus, vvhClient: VVHClient, roadLi
   def split(id: Long, splitMeasure: Double, existingValue: Int, createdValue: Int, username: String, municipalityValidation: (Int) => Unit): Seq[SpeedLimit] = {
     withDynTransaction {
       val newId = dao.splitSpeedLimit(id, splitMeasure, createdValue, username, municipalityValidation)
-      dao.updateSpeedLimitValue(id, existingValue, username, municipalityValidation)
-      Seq(loadSpeedLimit(id).get, loadSpeedLimit(newId).get)
+      val idUpdated = updateSpeedLimitValue(id, existingValue, username, municipalityValidation).get
+      Seq(loadSpeedLimit(idUpdated).get, loadSpeedLimit(newId).get)
     }
   }
 
   private def toSpeedLimit(persistedSpeedLimit: PersistedSpeedLimit): SpeedLimit = {
-    val roadLink = roadLinkServiceImplementation.getRoadLinkFromVVH(persistedSpeedLimit.linkId).get
+    val roadLink = roadLinkServiceImplementation.getRoadLinkFromVVH(persistedSpeedLimit.linkId, false).get
 
```

Comment:
Since we don't need to have the "toSpeedLimit" on the new update method we don't need to pass the "false" on "getRoadLinkFromVVH"